pangu-spacing.el

```diff
@@ -167,19 +167,25 @@ When you set t here, the space will be insert when you save file."
 ;;   [Emacs-diffs] trunk r115873: Fix misspelling of 'chinese' in rx.
 ;;
 ;; Url: http://lists.gnu.org/archive/html/emacs-diffs/2014-01/msg00049.html
+;;
+(defun pangu-spacing-chinese-two-byte-category ()
+  "Return the correct category name for Chinese two-byte characters."
+  (if (assq 'chinese-two-byte rx--categories)
```

Comment:
You also should require `rx` when byte compile, to inhibit error `Error: Symbol's value as variable is void: rx--categories`.

```
(eval-when-compile (require 'rx))
```

Reply:
Fixed😄